python/scriptloader/script_loader.py

```diff
@@ -60,7 +60,7 @@ class ScriptLoader(salobj.BaseCsc):
       Once the script is fully loaded, the loader will send the
       configuration specified in the ``load`` command.
     * Configuring the script causes it to output an estimate of its
-      duration and puts the script into the CONFIGURED state,
+      duration and puts the script into the `ScriptState.CONFIGURED` state,
```

Comment:
Only duration? Or all the information that the queue might need?

Reply:
Good catch. Left over from metadata implementation. I'll update it.